### DISCUSSION TITLE
CircleCI or GitHub Actions?

### DISCUSSION THREAD

**Discussion by warengonzaga** [warengonzaga/css-text-portrait-builder#48] [Brainstorm]
Still, thinking about what we should use for issue #9 a plain GitHub Actions or CicleCI? 😎

**Comment by decanTyme** [comment]
I have not used CircleCI before, but I'd say GH Actions should be enough for now since testing is not even implemented yet and it's just simple stylechecks. If it gets more complicated, we can then just copy whatever processes/steps over to CircleCI. Just my opinion tho.

**Reply by warengonzaga** [reply]
I would agree, let's do it.

**Comment by warengonzaga** [comment]
I'm no longer interested on adding automation for code checking.
